pkg/ddl/index.go

```diff
@@ -2616,10 +2643,125 @@ func (w *worker) executeDistTask(stepCtx context.Context, t table.Table, reorgIn
 			}
 		}
 	})
+
+	g.Go(func() error {
+		modifyTaskParamLoop(ctx, jobCtx.sysTblMgr, taskManager, done,
+			reorgInfo.Job.ID, taskID, lastConcurrency, lastBatchSize, lastMaxWriteSpeed)
+		return nil
+	})
+
 	err = g.Wait()
 	return err
 }
 
+// Note: we can achieve the same effect by calling ModifyTaskByID directly inside
```

Comment:
I didn't review most of DXF codes so I'm not sure what's the meaning od this function comment. Just to check: 

it says we only allow some functions to write to the system table of DXF, especially, we don't allow ADMIN ALTER DDL JOB to write because it's heavier to rollback on conflict?

Can you explain in this design, which functions are allowed to write the system table? I want to check if they still can conflict with each other.